test/com/engeto/CountryExtractorTest.java

```diff
@@ -0,0 +1,41 @@
+package com.engeto;
+
+import kong.unirest.GetRequest;
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import kong.unirest.json.JSONArray;
+import kong.unirest.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class CountryExtractorTest {
+    @Test
+    public void extractCountriesTest() {
+        GetRequest request = Unirest.get("http://jsonvat.com/");
```

Comment:
obecne na testovanie sa pouzivaju ine data, nez produkcne - produkcne data sa mozu zmenit (krajina zmeni zakon o DPH) a to by rozbil nas test. My chceme, aby nas test bol zeleny kym nas program funguje spravne, a to nemusi platit v tomto pripade. Najlepsie je vytvorit testovaci json s mock daty a nacist to zo suboru.